civic-tech-club/src/style/style.scss

```diff
@@ -0,0 +1,245 @@
+}
+
+.blob-mask-3{
+    mask-image: url('./assets/blob-3.svg');
+    -webkit-mask-image: url('./assets/blob-3.svg');
+}
+
+.blob-mask-4{
+    mask-image: url('./assets/blob-4.svg');
+    -webkit-mask-image: url('./assets/blob-4.svg');
+}
+
+.blob-mask-5{
+    mask-image: url('./assets/blob-5.svg');
+    -webkit-mask-image: url('./assets/blob-5.svg');
+}
+
+.content-container{
+    min-height:100vh;
+    text-align:left;
+}
+
+
+
+/*HOMEPAGE ==============================================*/
+
+/*NAVIGATION*/
+.nav-bg{
+    background-color:$primary-dark;
+}
+
+.nav-bg a{
+    font-family:$headings-font-family;
+    font-weight:600;
+}
+
+/*HERO*/
+.hero-text{
+    text-align:left;
+}
+.hero-image{
+    width: 100%;
+    aspect-ratio: 1 / 1;
+    min-width:300px;
+    object-fit:cover;
+}
+
+.hero-wrapper{
+    width: 100%;
+    min-width:320px;
+    aspect-ratio: 1 / 1;
+}
+
+.hero-stroke{
+    width: 100%;
+    min-width:320px;
+    aspect-ratio: 1 / 1;
+    background-color: $secondary-dark;
+}
+
+.hero-mask{
+    width: calc(100% - 20px);
+    min-width:300px;
+    aspect-ratio: 1 / 1;
+    top: 10px;
+    left: 10px;
+}
+
+
+/*PROJECT SECTION*/
+.hold-project-container{
+    background-color:$secondary-dark;
+}
+
+.featured-header{
+    color:white;
+    text-align:center;
+}
+
+.project-card{
+    border-radius:15px !important;
+}
+.project-card-image{
+    width:100%;
+    aspect-ratio: 3 / 2;
+    object-fit:cover;
+}
+.card-link{
+    border:5px solid transparent;
+    display:block;
+    height:100%;
+    color:unset;
+    text-decoration:none;
+    overflow:hidden;
+    border-radius:15px;
+}
+
+.card-link:hover{
+    border:5px solid $tertiary-light;
+}
+
+
+/*PROFILE SECTION*/
+
+.profile-image{
+    width: 50%;
+    aspect-ratio: 1 / 1;
+    min-width:200px;
+    object-fit:cover;
+}
+
+.profile-wrapper{
+    width: 50%;
+    min-width:210px;
+    aspect-ratio: 1 / 1;
+}
+
+.profile-stroke{
+    width: 50%;
+    min-width:210px;
+    aspect-ratio: 1 / 1;
+    background-color: $secondary-light;
+}
+
+.profile-mask{
+    width: calc(50% - 10px);
+    min-width:200px;
+    aspect-ratio: 1 / 1;
+    top: 5px;
+    left: 5px;
+}
+
+.holdProfile{
+    display:flex;
+    justify-content: center;
+    align-items:center;
+}
+
+/*FOOTER*/
+#contact{
+    color:white;
+    background-color:$primary-dark;
+}
+
+#contact ul{
+    margin-left:0;
+    padding-left:0;
+}
+#contact ul li{
+    list-style-type:none;
+    margin-left:0;
+    padding-left:0;
+}
+#contact a{
+    color:white;
+    text-decoration:none;
+}
+
+#contact a:hover{
+    text-decoration:underline;
+}
+
+.footer-icon{
+    width:30px;
+    height:auto;
+    fill:white;
+}
+
+/*PD CALENDAR=============================================*/
+
+/*BUDGET VISUALIZATION ===================================*/
+.budget-list li{
+    cursor:pointer;
+}
+
+.budget-list li:hover{
+    text-decoration:underline;
+}
+
+.graph{
+    height:90vh;
+}
+
+
+@import "bootstrap/scss/bootstrap";
```

Comment:
if we accidentally add classes that conflict with bootstrap's, importing it last will potentially wipe out our changes. We should likely move this to the top of the file, but underneath the other imports and variable declarations.

Reply:
This is resolved. I had a devil of a time with this, because trying to import the bootstrap.scss file was causing a ton of sass/bootstrap-react/vite warnings about deprecation. I managed to solve them by importing the bootstrap css from the dist folder, and then importing my scss file.

Reply:
I empathize :(. I've had to slog through that before too. Not fun.